setup.py

```diff
@@ -15,7 +15,7 @@
 
 setuptools.setup(
     name='wavefront-sdk-python',
-    version='1.2',
+    version='1.2.2',
```

Comment:
`1.2.2` is already taken by https://github.com/wavefrontHQ/wavefront-sdk-python/pull/42